### ISSUE TITLE
variable ${pageContext.request.contextPath} is displayed in input fied

### ISSUE THREAD

**Issue by Vad1mo** [adrianbk/swagger-springmvc-demo#14]
when running  http://localhost:8884/sdoc.jsp  ${pageContext.request.contextPath} is not getting resolved.

How can I set ${pageContext.request.contextPath} or make it resolvable?

![bildschirmfoto 2014-11-18 um 14 02 05](https://cloud.githubusercontent.com/assets/1492007/5087712/96d0793a-6f2b-11e4-92fd-ad46bad95242.png)

**Comment by bjconlan** [comment]
This is because spring boot (specifically spring-starter-web) doesn't define a jsp processor. By adding the appropriate `runtime 'org.apache.tomcat.embed:tomcat-embed-jasper'` dependency into my gradle project resolved the issue in my case. (runtime is the incorrect scope but gradle doesn't have 'provided' available by default)

The appropriate maven fragment is:
```xml
<dependency>
	<groupId>org.apache.tomcat.embed</groupId>
	<artifactId>tomcat-embed-jasper</artifactId>
	<scope>provided</scope>
</dependency>```

**Comment by fpeloi-bmj** [comment]
hi @bjconlan, I tried that but it didn't work (with gradle), did you have that issue with a SpringBoot application?

**Comment by bjconlan** [comment]
Yeah, are you still getting the same error? (ie evaluating the jstl el). If you are using intellij don't forget to refresh/reload the gradle build

**Comment by bjconlan** [comment]
Yeah, are you still getting the same error? (ie evaluating the jstl el). If you are using intellij don't forget to refresh/reload the gradle build
